src/steps/create-lead.ts

```diff
@@ -39,10 +41,13 @@ export class CreateLead extends BaseStep implements StepInterface {
 
   async executeStep(step: Step): Promise<RunStepResponse> {
     const stepData: any = step.getData().toJavaScript();
-
+    const dateTokenFormat = /\d{4}-\d{2}-\d{2}(?:.?\d{2}:\d{2}:\d{2})?/;
     for (const key in stepData.lead) {
       if (!isNaN(stepData.lead[key])) {
         stepData.lead[key] = parseFloat(stepData.lead[key]);
+      } else if (dateTokenFormat.test(stepData.lead[key])) {
+        const midnightReference = chrono.parseDate('midnight utc', new Date()).toISOString().split('T')[1];
+        stepData.lead[key] = stepData.lead[key].includes(midnightReference) ? moment(stepData.lead[key]).format('YYYY-MM-DD') : stepData.lead[key];
```

Comment:
- [ ] Since `midnight utc` will theoretically always resolve to `00:00:00`, it seems safe to hard-code that in, then we could avoid that extra dependency too.  It would also resolve potential issues where the provided date does not include milliseconds, or a UTC timezone indicator (e.g. `new Date().toISOString()` for UTC could be `2020-06-24T00:00:00.000Z`, but the input string could simply be `2020-06-24T00:00:00`)